packages/render/package.json

```diff
@@ -0,0 +1,85 @@
+{
+  "name": "@frames.js/render",
+  "version": "0.0.0",
+  "type": "module",
+  "main": "./dist/index.cjs",
+  "types": "index.d.cts",
+  "typesVersions": {
+    "*": {
+      "*": [
+        "./dist/*"
```

Comment:
have you tried to run `yarn build && yarn pack` in this folder and then upload it to https://arethetypeswrong.github.io ?

Reply:
will do

Reply:
types are fixed now, thanks for the heads up!